chia/wallet/cat_wallet/cat_wallet.py

```diff
@@ -208,6 +208,7 @@ async def create_wallet_for_cat(
 
         self.lineage_store = await CATLineageStore.create(self.wallet_state_manager.db_wrapper, self.get_asset_id())
         await self.wallet_state_manager.add_new_wallet(self, self.id())
+        self.wallet_state_manager.state_changed("wallet_created")
```

Comment:
Interestingly, `add_new_wallet` already calls `state_changed` - not sure if all the previous calls were also redundant. https://github.com/Chia-Network/chia-blockchain/blob/c209f886b1e8c30902c0b6e05884d55ad1c9591f/chia/wallet/wallet_state_manager.py#L1687